topostats/tracing/ordered_tracing.py

```diff
@@ -888,6 +889,8 @@ def ordered_tracing_image(
         Results containing the ordered_trace_data (coordinates), any grain-level metrics to be added to the grains
         dataframe, a dataframe of molecule statistics and a dictionary of diagnostic images.
     """
+    topoly_version = importlib.metadata.version("topoly")
+    print(f"Topoly version: {topoly_version}")
```

Comment:
Will this be permanent?

If so `LOGGER.debug()` might be appropriate and we could set up the tests to use a debug level in logging.

I will also address #896 now as its a small change and means you can SSH into GitHub runners when these fail and find out such information directly.

Reply:
Nah not permanent, just desperate debugging to see what version was being used

Reply:
Sylvia has just done this :)